tests/test_ecr/test_ecr_scanning_config.py

```diff
@@ -133,9 +137,120 @@ def test_put_registry_scanning_configuration():
             "repositoryArn": repo_arn,
             "repositoryName": repo_name,
             "scanOnPush": False,
-            "scanFrequency": "SCAN_ON_PUSH",
-            "appliedScanFilters": [
-                {"filter": f"{repo_name[:4]}*", "filterType": "WILDCARD"}
-            ],
+            "scanFrequency": "MANUAL",
+            "appliedScanFilters": [],
```

Comment:
Manually tested it against AWS. If a new scanning policy is pushed which does not match the existing repositories, they are overwritten with the default again.